rust/feature-flags/src/flag_matching.rs

```diff
@@ -91,7 +91,7 @@ impl GroupTypeMappingCache {
                 return Err(e);
             }
         };
-        self.group_types_to_indexes = mapping.clone();
+        self.group_types_to_indexes.clone_from(&mapping);
```

Comment:
I read up on this and it seems like `clone_from` is pretty generally a better operation.  Feels like I should replace this everywhere.  Thanks for bringing it up!